aero.minova.core.application.system.service/src/main/java/aero/minova/core/application/system/service/SetupService.java

```diff
@@ -0,0 +1,111 @@
+package aero.minova.core.application.system.service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+import aero.minova.core.application.system.sql.SystemDatabase;
+import lombok.val;
+
+@Service
+public class SetupService {
```

Comment:
Bitte die Methoden für das Aufspielen der SQL-Skripte als Extension in den SqlProcedureController einbauen,
damit man die Funktion ausführen kann.

Reply:
Bitte neue Funktionalität dokumentieren.
Dass muss nicht ausführlich gemacht werden,
da die Funktion insgesamt noch nicht fest ist.

Einfach beschreiben,
wie es getriggert wird und was das Ziel davon ist.

Reply:
> Bitte die Methoden für das Aufspielen der SQL-Skripte als Extension in den SqlProcedureController einbauen,
> damit man die Funktion ausführen kann.

In den Tests kriegen wir die dependencies ja aus der exampleDependencyList.txt.
Kriegen wir den String in Zukunft dann als value aus der inputTable der Prozedur?
Ist als Prozedurname `setup` ok?

Reply:
`setup` is als Name ok.
Die dependencies werden erstmal immer aus den Dateien ausgelesen
und nicht aus der inputTable ausgelesen,
da wir erstmal nur das Aufspielen der aller SQL-Skripte unterstützen.